spec/generators/active_outbox_generator_spec.rb

```diff
@@ -61,7 +61,7 @@ def change
               create_table :#{table_name}_outboxes do |t|
                 t.string :identifier, null: false, index: { unique: true }
                 t.string :event, null: false
-                t.string :payload
```

Comment:
Add a specific MySQL test, and change the name for this one to be 'when it is a sqlite migration'